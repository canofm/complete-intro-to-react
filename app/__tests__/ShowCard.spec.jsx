import React from 'react';
import { shallow } from 'enzyme';
import ShowCard from '../components/ShowCard';

describe('ShowCard component', () => {
  it('should renders correctly', () => {
    const show = {
      title: 'A title',
      description: 'A description',
      poster: 'A poster',
      year: 'A year'
    };

    const component = shallow(<ShowCard {...show} />);
    expect(component).toMatchSnapshot();
  });
});
