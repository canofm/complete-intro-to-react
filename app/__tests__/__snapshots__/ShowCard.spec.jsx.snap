// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowCard component should renders correctly 1`] = `
<styled.div>
  <styled.img
    alt="A title Show Poster"
    src="/public/img/posters/A poster"
  />
  <div>
    <h3>
      A title
    </h3>
    <h4>
      (
      A year
      )
    </h4>
    <p>
      A description
    </p>
  </div>
</styled.div>
`;
